graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -360,6 +381,35 @@ abstract class BaseDataTypeGenerator(
 
     private fun addField(fieldDefinition: Field, javaType: TypeSpec.Builder) {
         addFieldWithGetterAndSetter(fieldDefinition.type, fieldDefinition, javaType)
+        // Generate for all fields except non-nullable without default values
```

Comment:
nitpick: I think the comment would be clearer if it was stated as what we want to generate the fields for, e.g.:

> Generate for all nullable fields without any defaults

Reply:
Updated 👍